pkg/printers/internalversion/printers.go

```diff
@@ -1002,7 +1003,11 @@ func printStatefulSet(obj *apps.StatefulSet, options printers.PrintOptions) ([]m
 	row.Cells = append(row.Cells, obj.Name, desiredReplicas, currentReplicas, createTime)
 	if options.Wide {
 		names, images := layoutContainerCells(obj.Spec.Template.Spec.Containers)
-		row.Cells = append(row.Cells, names, images)
+		selector, err := metav1.LabelSelectorAsSelector(obj.Spec.Selector)
```

Comment:
why not use `FormatLabelSelector` like job, daemonSet... ?

Reply:
Both methods can be implemented, but this requires format conversion

Reply:
ok, seems selector of `StatefulSet` can not be empty.